### DISCUSSION TITLE
How to Enable Drag and Drop

### DISCUSSION THREAD

**Discussion by khalidaboelmagd** [PeterStrick/ViVeTool-GUI#14] [General]
Hello, 
which hidden features to enable so that I can have the old drag and drop again?

**Comment by PeterStrick** [comment]
Do you mean the Drag and Drop for the Taskbar? It should be enabled by default on the latest Dev Builds without needing ViVeTool

**Comment by alesimula** [comment]
@PeterStrick I'm on Windows 11 BETA version 22622.440 and still don't have this feature, it was either disabled or removed

Either way, I'd like to know the answer to this question too

**Comment by Seath1337** [comment]
Same here... on 22622.450...few beta builds ago, it was there...but got removed for some reason
